src/System.Management.Automation/engine/COM/ComDispatch.cs

```diff
@@ -19,7 +21,7 @@ internal interface IDispatch
         int GetTypeInfoCount(out int info);
 
         [PreserveSig]
-        int GetTypeInfo(int iTInfo, int lcid, out COM.ITypeInfo ppTInfo);
+        int GetTypeInfo(int iTInfo, int lcid, out COM.ITypeInfo? ppTInfo);
```

Comment:
`ITypeInfo** ppTInfo`
The requested type information object.  This will be null if iTInfo is not 0 and the method returns DISP_E_BADINDEX.

Reply:
Could you please look https://github.com/dotnet/winforms/blob/389f909e1836947321cd0b3e104894eaac8cdbc9/src/System.Windows.Forms.Primitives/src/Interop/OleAut32/Interop.IDispatch.cs? They do not consider this as nullable.

Reply:
>ppTInfo: MUST be set to reference an instance of an ITypeInfo server that corresponds to the default nonsource interface of the coclass implementing IDispatch (see section 2.2.49.8). MUST refer to the partner dispinterface if the default nonsource interface is a dual interface. MUST be set to **NULL** if the coclass does not specify a default nonsource interface.

I'm not sure WinForms has annotated these. At least not correctly.